Variable_Project/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿//Develop a basic c# console application that computes and shows the grade entered by the user and it should be in lower case and should not pass the range of f grade.
+// Show the integer value of the grade.
+Console.Write("Enter the grade:");
+char gradeScore = char.Parse(Console.ReadLine());
```

Comment:
Why are you using a char parse. Look for a way to cast this to either an integer or a char